js/app/modules/card/knowledgeDocument.js

```diff
@@ -82,6 +82,12 @@ const KnowledgeDocument = new Module.Class({
         'next-card': GObject.ParamSpec.object('next-card',
             'Next Card', 'Next Card',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY, Gtk.Widget),
+        /**
+         * Property: nav-content
+         */
+        'nav-content': GObject.ParamSpec.object('nav-content',
```

Comment:
I'm not too excited about this implementation. It seems like it allows you to specify both `next-card` and `nav-content` on top of each other?

Consider refactoring `next-card` and `previous-card` so that they can also be expressed as a content group.